src/Platforms/SqlitePlatform.php

```diff
@@ -328,14 +330,47 @@ protected function _getCommonIntegerTypeDeclarationSQL(array $column)
         return ! empty($column['unsigned']) ? ' UNSIGNED' : '';
     }
 
+    /**
+     * Disables schema emulation.
+     *
+     * Schema emulation is enabled by default to maintain backwards compatibility.
+     * Disable it to opt-in to the behavior of DBAL 4.
+     *
+     * @deprecated Will be removed in DBAL 4.0.
+     */
+    public function disableSchemaEmulation(): void
+    {
+        $this->schemaEmulationEnabled = false;
+    }
+
+    /**
+     * Enables schema emulation.
+     *
+     * Schema emulation is enabled by default to maintain backwards compatibility.
+     * Disable it to opt-in to the behavior of DBAL 4.
```

Comment:
Why both the comments for `disableSchemaEmulation()` and `enableSchemaEmulation()` suggest disabling schema emulation? Please either make these comments reasonable or remove them.

Reply:
Please propose some wording or let me know to remove them. Or maybe keep it only above `disableSchemaEmulation()` method?